apps/server/src/modules/board/repo/board-do.repo.ts

```diff
@@ -18,6 +19,15 @@ export class BoardDoRepo {
 		return domainObject;
 	}
 
+	async findByClassAndId<T extends AnyBoardDo>(doClass: ConstructorOf<T>, id: EntityId, depth?: number): Promise<T> {
```

Comment:
I also played around with it a little bit yesterday and I think we could save the import and definition oft that type (along with disabling of eslint for that line) by using typescript's built-in functionality for defining a class type: https://www.typescriptlang.org/docs/handbook/2/generics.html#using-class-types-in-generics

So this line would be: 
`async findByClassAndId<S, T extends AnyBoardDo>(doClass: { new (props: S): T }, id: EntityId, depth?: number): Promise<T> `

Reply:
Oh, that's cool. We should do that 👍